README.md

```diff
@@ -52,6 +52,33 @@ brew install libmagic
 brew link libmagic  (if the link is already created is going to fail, don't worry about that)
 ```
 
+Additionally, problems have been reported on Windows when installing Submit4DN
+inside a virtual environment, due to `aws` trying to use the global python instead
+of the python inside the virtual environment.
+
+The workaround, then, because it’s actually OK if `aws` doesn’t use the python
+inside the virtual environment, is to just install `awscli` in the global
+environment before entering the virtual environment. Or if you discover the
+problem after you’re in, then go outside, install `awscli`, and re-enter the
+virtual environment.
+
+```
+deactivate
+pip install awscli
+venv\scripts\activate
```

Comment:
Do we suggest an env name for users to use? I used `venv` here when responding to the ticket, but we should make sure we pick a consistent name throughout the doc.

Reply:
This is the only place in the doc where it is mentioned. I clarified that this is just a placeholder and the user should pick the appropriate virtual environment name.